src/components/NoteItem.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <section class="item-box item-note-list">
+    <div class="item-box__media">
+      <img v-bind:src="noteItem.thumbUrl" />
```

Comment:
My component uses abbreviations, is it better to use `v-bind` explicitly?
`<img :src="noteItem.thumbUrl" />`

Reply:
No one is better, but v-bind is verbose, we'll edit.